task-sdk/pyproject.toml

```diff
@@ -138,13 +138,15 @@ codegen = [
     "datamodel-code-generator[http]==0.28.2",
     "openapi-spec-validator>=0.7.1",
     "svcs>=25.1.0",
-    "ruff==0.11.13",
+    "ruff==0.12.1",
     "rich>=12.4.4",
 ]
 dev = [
     "apache-airflow-providers-common-sql",
     "apache-airflow-providers-standard",
     "apache-airflow-devel-common",
+    "pandas>=2.1.2; python_version <\"3.13\"",
+    "pandas>=2.2.3; python_version >=\"3.13\""
```

Comment:
@potiuk What did adding this fix? Nothing in task-sdk directly depends on pandas, so I'm not immediately sure why we need this.

Reply:
Ohh I missed the fact this was in dev group. 🤦🏻 

(The only reason I asked was this seemed to be causing some issues building when it was backported. I didn't really dig in to it in much detail though)